dvc/repo/add.py

```diff
@@ -1,11 +1,21 @@
 import os
 from dvc.stage import Stage
+from dvc.utils import walk_files
 
 
-def add(repo, fname, recursive=False, no_commit=False):
-    fnames = _collect_filenames_to_add(repo, fname, recursive)
+def add(repo, target, fname=None, recursive=False, no_commit=False):
```

Comment:
Please don't change the order of arguments. New arguments should be appended to the end.